packages/thirdweb/src/react/web/wallets/in-app/InAppWalletFormUI.tsx

```diff
@@ -43,7 +48,9 @@ export const InAppWalletFormUI = (props: InAppWalletFormUIProps) => {
   const locale = props.locale;
   const { chain, client, connectModal } = useConnectUI();
   const { done, wallet } = props;
-  const setData = useSetSelectionData();
+  const setData = useSetSelectionData() as (
+    value: InAppWalletSelectUIState,
+  ) => void;
```

Comment:
Because it's not typed - the `selectionData` is generic data in context shared by all wallets - it is used for sending data from the wallet "selection screen" to wallet "connection screen" 

Right now only the inApp wallet stores such data, so we can could type it, but if in future we want to expose this context for custom wallets - we will have to revert this

Reply:
okay for now but mental note that we need to walk through the react context structure :)